src/pages/patientView/PatientViewPage.tsx

```diff
@@ -335,7 +336,7 @@ export default class PatientViewPage extends React.Component<IPatientViewPagePro
                                         myCancerGenomeData={patientViewPageStore.myCancerGenomeData}
                                         hotspots={patientViewPageStore.indexedHotspotData}
                                         cosmicData={patientViewPageStore.cosmicData.result}
-                                        oncoKbData={patientViewPageStore.oncoKbData.result}
+                                        oncoKbData={patientViewPageStore.oncoKbData.error ? ONCOKB_ERROR : patientViewPageStore.oncoKbData.result}
```

Comment:
I think, what we really should do here is to pass the entire `patientViewPageStore.oncoKbData`  to the child component, instead of just the `result`. Or at least we should pass both the `result` and the `status`. That way, we don't need to derive status from the `result`.